src/schemata/runSettingsSchema.ts

```diff
@@ -0,0 +1,13 @@
+import * as S from '@effect/schema/Schema';
+
+export const runSettingsSchema = S.union(
+	S.struct({
+		dryRun: S.optional(S.literal(false)).withDefault(() => false),
+	}),
+	S.struct({
+		dryRun: S.literal(true),
```

Comment:
```suggestion
		dryRun: S.literal(true),
```

Is `S.literal(true)` correct here? Just double checking.

Reply:
yes, that is a boolean literal.